goodtablesio/tests/plugins/github/test_blueprint.py

```diff
@@ -0,0 +1,59 @@
+# pylama:skip=1
+import json
+from unittest.mock import patch
+from goodtablesio.tasks import app as celery_app
+from goodtablesio.plugins.github.blueprint import _get_owner_repo_sha
+
+
+# Tests
+
+# High-level test for now fails because of db session error
```

Comment:
I think this test is pretty important to have because it checks the whole system top-down.